rootfs/standard/usr/bin/mynode_post_upgrade.sh

```diff
@@ -424,6 +427,11 @@ if [ "$CURRENT" != "$WHIRLPOOL_UPGRADE_URL" ]; then
     sudo rm -rf *.jar
     sudo -u bitcoin wget -O whirlpool.jar $WHIRLPOOL_UPGRADE_URL
 
+    echo "$WHIRLPOOL_HASH  whirlpool.jar" > WHIRLPOOL_SHASUM
```

Comment:
Is this saving this text to a file named "WHIRLPOOL_SHASUM" or was it meant to be a variable? If a file, can we put it in /tmp/.... and something with no caps to prevent confusion?

Reply:
Good point. I'll use a file in /tmp with lower-case